indexes/channels.js

```diff
@@ -0,0 +1,109 @@
+const bipf = require('bipf')
+const pull = require('pull-stream')
+const pl = require('pull-level')
+const jsonCodec = require('flumecodec/json')
+const Plugin = require('ssb-db2/indexes/plugin')
+
+const isFeed = require('ssb-ref').isFeed
```

Comment:
not needed

Reply:
isFeed could do, but jsonCodec and Plugin are used further down.

Reply:
Right, the comment was only for isFeed